docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -117,7 +117,7 @@ The [Camunda provided module](https://github.com/camunda/camunda-tf-eks-module)
 
 ```hcl
 module "eks_cluster" {
-  source = "github.com/camunda/camunda-tf-eks-module/modules/eks-cluster"
+  source = "git::https://github.com/camunda/camunda-tf-eks-module//modules/eks-cluster.git?ref=2.0.0"
```

Comment:
the `.git` part doesn't work due to the sub directory.

as mentioned last time, either
```
git::https://github.com/camunda/camunda-tf-eks-module//modules/eks-cluster?ref=2.0.0
github.com/camunda/camunda-tf-eks-module//modules/eks-cluster?ref=2.0.0
```

Reply:
or
```
git::https://github.com/camunda/camunda-tf-eks-module.git//modules/eks-cluster?ref=2.0.0
```

would work if you want the `.git`